.yamato/_abv.yml

```diff
@@ -43,10 +37,21 @@ all_project_ci_nightly_2020.2:
          rerun: on-new-revision
       -  path: .yamato/all-vfx_urp.yml#Nightly_VFX_URP_2020.2
          rerun: on-new-revision
+      -  path: .yamato/all-urp_performance_boatattack.yml#PR_URP_Performance_BoatAttack_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/all-urp_performance_lostcrypt.yml#PR_URP_Performance_LostCrypt_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/all-urpupdate.yml#PR_URPUpdate_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/_projectcontext.yml#all_package_ci_project_2020.2
+         rerun: on-new-revision
     triggers:
         recurring:
           -  branch: 10.x.x/release
              frequency: 1 * * ?
+all_project_ci_weekly_2020.2:
```

Comment:
Feel free to remove it if it's possible via the metafiles (it's useless because it has no dependencies). Not a blocker though.

Reply:
I'll leave it for now!